pyaedt/modeler/PrimitivesNexxim.py

```diff
@@ -1929,6 +1929,8 @@ def create_component_from_spicemodel(self, model_path, model_name=None, location
             Path to .lib file.
         model_name : str, optional
             Model name to import. If `None` the first subckt in the lib file will be placed.
+        create_component: bool
+            True: create spice model component. False: only import spice model
```

Comment:
```suggestion
        create_component: bool, optional
            If set to ``True``, create a spice model component. Otherwise, only import the spice model.
```